mne/io/base.py

```diff
@@ -2258,9 +2259,8 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
         if split_naming == 'neuromag':
```

Comment:
Now that I see it, I might even do something like: 

```py
def get_split_name(base, part_idx, ext, format=None):
    """
    return the proper split name for different formats [neuromag, bids]
    param..
    """

    if format == 'neuromag':
       out = '{base}s-{part}d{ext}s'.format(base=base, part=part_idx, ext=ext)
    elif split_naming == 'bids':
       out = _construct_bids_filename(base, ext, part_idx)
    else:
        raise something

    return out
```

Reply:
And I would test it separately rather than test it in `test_split_files`. So that if I want to see which are the valid formats I can just check the test.

Reply:
I didn't want to make this a function because there is the strong possibility that we might deprecate the enforcement of the neuromag naming conventions. I think that this function would not be used in any other case than this specific one.